tests/unit/cmorizers/test_datasets_info.py

```diff
@@ -0,0 +1,48 @@
+import os
+import datetime
+import yamale
+import yaml
+
+yaml_folder = os.path.abspath(os.path.join(
+    os.path.dirname(__file__), '../../..', 'esmvaltool/cmorizers/data'))
+
+recipes_folder = os.path.abspath(os.path.join(
+    os.path.dirname(__file__), '../../..', 'esmvaltool/recipes'))
```

Comment:
Instead of obtaining the path like this, which is relative to the tests, you could do something like
```python
yaml_folder = pathlib.Path(esmvaltool.cmorizers.__file__).parent
```
that should fix the tests